frontend.sh

```diff
@@ -49,7 +49,7 @@ install() {
     echo "Installing frontend development dependencies…"
     yarn install
 
-    # Protractor = JavaScript acceptance testing framework.
+    # Protractor = JavaScript browser testing framework.
```

Comment:
To investigate: Is Protractor needed by Cypress?

Reply:
No Protractor is not needed by Cypress